client/landing/gutenboarding/hooks/use-site-title.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { useDispatch } from '@wordpress/data';
+import { useEffect } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { STORE_KEY as ONBOARD_STORE } from '../stores/onboard';
+import usePodcastTitle from './use-podcast-title';
+
+export default function useSiteTitle(): void {
+	const { setSiteTitle } = useDispatch( ONBOARD_STORE );
+	const podcastTitle = usePodcastTitle();
+
+	useEffect( () => {
+		if ( podcastTitle && podcastTitle.length > 1 ) {
```

Comment:
I think just `if ( podcastTitle )` will do, since an empty string would evaluate to false.

Reply:
I'd added that check because of this selector and note:
https://github.com/Automattic/wp-calypso/blob/8f80f415404c104b1e81d7fc07c177df0548d1ad/client/landing/gutenboarding/stores/onboard/selectors.ts#L30
But it looks like it isn't necessary here, since we don't show the "next" button if that selector doesn't return true.